Dockerfile.api

```diff
@@ -34,4 +38,4 @@ RUN npm install --workspace @stela/api
 ENV PORT=80
 EXPOSE 80
 
-CMD node packages/api/dist/index.js
+CMD node -r newrelic packages/api/dist/index.js
```

Comment:
i'm curious about the use of the cli require flag. did you have issues importing newrelic from node_modules?

Reply:
That was left over from some troubleshooting I was doing; it did not turn out to be necessary. Thanks for catching it!